buildSrc/subprojects/parser/src/main/java/org/apache/calcite/buildtools/parser/DialectTraverser.java

```diff
@@ -143,9 +143,15 @@ private void traverse(
       String fileName = f.getName();
       if (f.isFile() && fileName.endsWith(".ftl")) {
         try {
-          String fileText = new String(Files.readAllBytes(f.toPath()),
+          Path absoluteFilePath = f.toPath();
```

Comment:
nit: This is getting to be a lot of contents in this try/catch block. Can we split out some things to not be in the try/catch?

Reply:
Will do